src/components/cookies/CookieConsent.tsx

```diff
@@ -123,6 +108,24 @@ export const CookieConsent = () => {
           </Script>
         </>
       )}
+      {cclwAnalyticsAllowed && (
+        <>
+          <Script async src="https://www.googletagmanager.com/gtag/js?id=UA-153841121-2" />
+          <Script id="cclw-google-analytics" strategy="afterInteractive">
+            {`
+              (function(w,l,i) {
+                w[l] = w[l] || [];
+                function gtag(){w[l].push(arguments);}
+                gtag('js', new Date());
+                gtag('config', i);
+              })(window, "dataLayer", "UA-153841121-2");
+            `}
+          </Script>
+        </>
+      )}
+      {theme === "cclw" && (
+        <Script id="cclw-plausible" defer data-domain="climate-laws.org" strategy="afterInteractive" src="https://plausible.io/js/script.js" />
```

Comment:
What will this line do?

Reply:
loads a third party script called plausible if the theme is set to cclw.

only reason I am keeping it in the cookie consent file is just so it is with all the other 3rd party tracking / library stuff